components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -162,7 +162,9 @@ be lost if SCREAM_HACK_XML is not enabled.
 
     <!-- Surface coupling (import and export) -->
     <sc_import inherit="atm_proc_base"/>
-    <sc_export inherit="atm_proc_base"/>
+    <sc_export inherit="atm_proc_base">
+      <prescribed_export_control_file type="string">NONE</prescribed_export_control_file>
```

Comment:
I know you're still in dev phase, but to avoid rethinking this later when code is bigger: I would prefer keeping these export options _inside_ this XML section, rather than delegating to another input file.

Reply:
I thought about this too and hopefully set up the code so if we ultimately decide to go this route it's not a big change.  For now I went w/ a separate file because I still don't know how complex the parameters will be for the case where we will force the export values w/ data from a set of files.

Reply:
okay, I changed it.